tests/baselines/reference/tsserver/typingsInstaller/discover-typings-should-return-node-for-core-modules.js

```diff
@@ -3518,7 +3486,7 @@ ts.JsTyping.discoverTypings::
         "enable": true
       },
       "unresolvedImports": [
-        "node:zlib",
+        "node:quic",
```

Comment:
I was very confused why everything in this list got changed until I realized it's because one module was removed and one was added, so the diff makes it look like the code shifted